src/community/who-uses-dart.md

```diff
@@ -109,8 +109,8 @@ Google internal sales tool
 [WorkTrail](https://worktrail.net)
 : Time tracking app.
 
-[Angular](https://github.com/angular/angular.dart)
-: Angular 2 for Dart is a port of the popular framework to Dart.
+[AngularDart](https://github.com/angular/angular.dart)
```

Comment:
Let's delete this one. It's actually pointing to the original AngularDart implementation (1.x), which has been replaced by dart-lang/angular (2+).